### PR TITLE
Add cloud front function updates for version 1.2.0, makes swagger-ui work on AWS

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
The cloudfront function used to add headers to the incoming request has been updated and tested.

## ♻️ Related Issues

completes #331

### CODE REVIEW

**Comment by tloubrieu-jpl** [comment]
Sorry I messed with my rebase, and for some reason we see updates on the github actions in my PR.

File: .github/workflows/stable-cicd.yaml
Comment (alexdunnjpl) [inline .github/workflows/stable-cicd.yaml:1]: nit: if you can be bothered to revert this change, then squash the change commit and revert commit with another commit (thereby making the change/revert cancel each other out and become invisible), that's preferable, but not important

File: .github/workflows/unstable-cicd.yaml
Comment (alexdunnjpl) [inline .github/workflows/unstable-cicd.yaml:1]: nit: if you can be bothered to revert this change, then squash the change commit and revert commit with another commit (thereby making the change/revert cancel each other out and become invisible), that's preferable, but not important

File: service/terraform/aws/api_uri_rewrite.js [hunk 1]
```diff
@@ -14,74 +17,55 @@ var VER_IDX = 3
...
+        // split the uri
         var uriParts = incomingUri.split("/");
 
-        // at a minimum service, version and command are required (i.e. there are non-empty components in the URI after
-        // "/api/search/<version>") in order consider a rewrite of the URI, otherwise fall through w/o changes
-        if (uriParts.length > CMD_IDX && uriParts[CMD_IDX].trim() != "") {
-            if (debug) console.log("incoming URI includes a command")
-
+        // at a minimum service, version and command are required, otherwise fall through w/o changes
+        //if (uriParts.length > CMD_IDX && uriParts[CMD_IDX].trim() != "") {
```
Comment (alexdunnjpl) [inline service/terraform/aws/api_uri_rewrite.js:41]: nit: remove if not required

File: service/terraform/aws/api_uri_rewrite.js
Comment (alexdunnjpl) [inline service/terraform/aws/api_uri_rewrite.js:1]: Rubber-stamping these changes

**Review by alexdunnjpl — APPROVED** [review]
Approved with nitpicks
